Sources/View/ReusableView.swift

```diff
@@ -20,15 +20,19 @@ public extension ReusableView where Self: UITableViewCell {
     init() {
         self.init(style: .default, reuseIdentifier: Self.reuseIdentifier)
 
-        setupLayout()
+        setupSubviews()
```

Comment:
I guess that being both grammatically correct and used by Apple are strong arguments to change it 😛

Let's try and do the **Right Thing ®** 😂